docker/federation/start-local-instances.bash

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 set -e
 
-sudo docker-compose down
-
-sudo docker build ../../ --file ../Dockerfile -t lemmy-federation:latest
+sudo docker compose down
 
 for Item in alpha beta gamma delta epsilon ; do
   sudo mkdir -p volumes/pictrs_$Item
   sudo chown -R 991:991 volumes/pictrs_$Item
 done
 
-#sudo docker-compose pull --ignore-pull-failures || true
-sudo docker-compose up
+#sudo docker compose pull --ignore-pull-failures || true
```

Comment:
Should I remove this? It's already commented out anyway.

Reply:
Yes you can remove it.